persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/spring/data/jpa/query/UserRepositoryIntegrationTest.java

```diff
@@ -40,46 +40,42 @@ public void whenFindAllActiveUsersNativeThenAllActiveFound() {
     @Test
     public void whenFindAllSortedByNameThenAllSorted() {
         List<User> allUsersSortedByName = userRepository.findAll(Sort.by(Sort.Direction.ASC, "name"));
-        assertThat(allUsersSortedByName)
-                .extracting("name")
+        assertThat(allUsersSortedByName).extracting("name")
                 .containsSequence("Bob", "Cindy", "John");
     }
 
     @Test
     public void whenFindAllSortedByNameLengthThenException() {
-        assertThatThrownBy(() -> userRepository.findAll(Sort.by("LENGTH(name)")))
-                .isInstanceOf(PropertyReferenceException.class);
+        assertThatThrownBy(() -> userRepository.findAll(Sort.by("LENGTH(name)"))).isInstanceOf(PropertyReferenceException.class);
     }
 
     @Test
     public void whenFindAllUsersSortedByNameThenAllSorted() {
         List<User> allUsersSortedByName = userRepository.findAllUsers(Sort.by(Sort.Direction.ASC, "name"));
-        assertThat(allUsersSortedByName)
-                .extracting("name")
+        assertThat(allUsersSortedByName).extracting("name")
                 .containsSequence("Bob", "Cindy", "John");
     }
 
     @Test
     public void whenFindAllUsersSortedByNameLengthThenAllSorted() {
         List<User> allUsersSortedByName = userRepository.findAllUsers(JpaSort.unsafe("LENGTH(name)"));
-        assertThat(allUsersSortedByName)
-                .extracting("name")
+        assertThat(allUsersSortedByName).extracting("name")
                 .containsSequence("Bob", "John", "Cindy");
     }
 
     @Test
     public void whenFindAllUsersWithPaginationThenPaginated() {
         Page<User> page = userRepository.findAllUsersWithPagination(PageRequest.of(0, 1));
-        assertThat(page.stream().map(User::getId))
-                .hasSize(1)
+        assertThat(page.stream()
```

Comment:
The original formatting is more readable I think. Let's keep it.

Reply:
done